config/config.go

```diff
@@ -21,8 +21,9 @@ import (
 )
 
 const (
-	OperatorName      string = "configure-alertmanager-operator"
-	OperatorNamespace string = "openshift-monitoring"
+	OperatorName       string = "configure-alertmanager-operator"
+	OperatorNamespace  string = "openshift-monitoring"
+	EnableOLMSkipRange        = "true"
```

Comment:
Is this supposed to be a string or bool?

Reply:
Doesn't really matter as it's only used by shell commands
https://github.com/openshift/configure-alertmanager-operator/blob/master/boilerplate/openshift/golang-osd-operator/project.mk#L14 

With `EnableOLMSkipRange        = "true"` 
```
$ gsed -n 's/.*EnableOLMSkipRange .*"\([^"]*\)".*/\1/p' config/config.go  
true
```
With `EnableOLMSkipRange string = "true"`
```
$ gsed -n 's/.*EnableOLMSkipRange .*"\([^"]*\)".*/\1/p' config/config.go
true
```

I added `string` just to make the code clean